packages/dapp/src/components/DualWidget/DualWidget.tsx

```diff
@@ -56,35 +65,90 @@ export function DualWidget() {
     }
   }, [activeTab, onChangeTab, starterVariant, starterVariantUsed]);
 
+  const handleGetStarted = async () => {
+    setShowFadeOut(true);
+    await setTimeout(() => {
+      setShowWelcome(false);
+    }, 300);
+  };
+
+  const isWalletConnected = useMemo(() => {
+    const activeWallet = JSON.parse(
+      localStorage.getItem(LOCAL_STORAGE_WALLETS_KEY),
+    );
+    return activeWallet?.length > 0 || false;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [account.isActive]);
+
   useEffect(() => {
     getActiveWidget();
   }, [getActiveWidget, starterVariant, activeTab]);
 
   return (
-    <Grid
-      justifyContent="center"
-      alignItems="center"
-      container
-      sx={{
-        overflowX: 'hidden',
-      }}
+    <WelcomeWrapper
+      showWelcome={!isWalletConnected && showWelcome}
+      showFadeOut={!isWalletConnected && showFadeOut}
```

Comment:
It is very repetetive and could be DRYer. By just combining both into a const which replaces them, would that be a better solution?